src/core/app/Page.ts

```diff
@@ -29,7 +29,13 @@ export default class Page extends EventEmitter {
     this.options = options;
 
     this.page.on('console', (data: puppeteer.ConsoleMessage) => {
-      consoleHandler(data.type(), data.text());
+      if (typeof data.type === 'function') {
+        consoleHandler(data.type(), data.text());
+      } else {
+        // it IS a string, by fact. Type definitions are wrong
+        // @ts-ignore
+        consoleHandler((data.type as string), (data.text as string));
```

Comment:
According to the puppeteer source code (https://github.com/GoogleChrome/puppeteer/blob/a310d57d847b008dba3b0772085840dde3343c64/lib/Page.js#L456) this IS a string.